Sources/Fluent/Entity/Entity.swift

```diff
@@ -82,6 +84,10 @@ extension Entity {
     public static var name: String {
         return String(describing: self).lowercased()
     }
+    
+    public static var idType: Schema.Field.KeyType? {
```

Comment:
should this be optional? Seems like we're defaulting to `.int` all over the place, maybe here could just be the point of default. Not sure of further implications somewhere else right now